src/index.js

```diff
@@ -5,107 +5,209 @@ import './view/css/style.css';
 import home from './view/home';
 import footer from './view/footer';
 
-let projectArr = [];
+// import controller modules
+import Project from './controller/project';
+import Todo from './controller/todo';
 
-const main = document.querySelector('#content');
+// import model
+import LocalSave from './model/localSave';
+
+const projectList = LocalSave;
+let currentProjectIndex = null;
 
+// loads home
+const main = document.querySelector('#content');
 main.insertAdjacentHTML('beforeEnd', '<div class="activeInfo"></div>');
 main.insertAdjacentHTML('beforeEnd', footer);
-
 const activeInfo = document.querySelector('.activeInfo');
 activeInfo.innerHTML = home;
 
+// save to local storage
+function saveLocal() {
+  localStorage.setItem('projectList', JSON.stringify(projectList));
+}
 
-const project = (pName) => {
-  const name = pName;
-  let taskArr = [];
+// add project to projectList, save local and reload
+function addProject() {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright